docs/how-to-guides/landscape-installation-and-set-up/install-on-fips-compliant-machines.md

```diff
@@ -1,9 +1,11 @@
 (how-to-install-fips-compliant)=
-# How to install on FIPS-compliant machines
+# How to install a FIPS-compliant Landscape Server
```

Comment:
I don't think this is precise though - (AFAIK) it's not _Landscape_ that's actually FIPS-certified.

If you prefer it to be more clear that it's for Landscape Server in the title, how about "How to install Landscape Server on FIPS-compliant machines"? Would that solve what you're getting at?